src/logic/calculate.js

```diff
@@ -0,0 +1,20 @@
+import operate from './operate';
+
+const calculate = (obj, buttonName) => {
+  const calcData = { obj };
+  if (buttonName === '+/-') {
+    calcData.total *= -1;
+    calcData.next *= -1;
```

Comment:
- [ ] Rather than retrieving the values of `obj` this way rather do this:
```
const {total, next} = obj
```
- [ ] It's best to give descriptive names to variables and function parameters, so kindly rename `obj` to `calcData`.